pkgs/extension_discovery/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: extension_discovery
+description: >-
+  Discovery of packages that provide extensions for your package.
+version: 1.0.0-wip
+publish_to: none
+repository: https://github.com/dart-lang/tools/tree/main/pkgs/extension_discovery
+
+dev_dependencies:
+  dart_flutter_team_lints: ^1.0.0
+  lints: ^2.0.0
+  test: ^1.21.0
+  test_descriptor: ^2.0.1
+
+environment:
+  sdk: ^3.0.0
```

Comment:
do all these dependencies already exist in the Dart SDK? or would they be acceptable to bring in? package`:extension_discovery` will be depended upon by `package:devtools_shared`, which is a dep of `package:dds` and therefore the Dart SDK.

Reply:
I see `[lints_rev](https://github.com/dart-lang/sdk/blob/main/DEPS#L155)`, `[test_rev](https://github.com/dart-lang/sdk/blob/main/DEPS#L178)`, and `[test_descriptor_rev](https://github.com/dart-lang/sdk/blob/main/DEPS#L179)` in DEPS, but I do not see `dart_flutter_team_lints`.

Reply:
It's in there :) It comes in as part of the dart-lang/ecosystem repo dep.